### DISCUSSION TITLE
github automation

### DISCUSSION THREAD

**Discussion by nexovec** [esin/esin#24] [Guestbook]
Hi, I am genuinely wondering what data you're collecting on github and why it's necessary to follow so many people.

**Comment by esin** [comment]
@nexovec no data is collected 

> why it's necessary to follow so many people.

because I can :)

**Comment by nexovec** [comment]
That's tough to believe, but cool if true :D Did you write the crawler yourself or is it publicly accessible somewhere? (not gonna do it myself regardless, not why I'm asking)

**Reply by esin** [reply]
I wrote this tool, for fun, some years ago, you can check my repo: https://github.com/esin/ghfollow

It has simple algo for following new users

**Reply by nexovec** [reply]
That's a great project, thanks for the info. I'm quite surprised github doesn't mind this :D. Happy hacking!
